recordedit/recordEdit.app.js

```diff
@@ -161,9 +161,7 @@
                             var column, value;
 
                             // $rootScope.tuples is used for keeping track of changes in the tuple data before it is submitted for update
-                            // We don't want to mutate the actual tuples associated with the page returned from `reference.read`
-                            // The submission data is copied back to the tuples object before submitted in the PUT request
-                            $rootScope.tuples = angular.copy(page.tuples);
+                            $rootScope.tuples = [];
```

Comment:
Side note: seems off that these things are stored directly in `$rootScope` but that doesn't need to be addressed by this PR.